model/src/test/kotlin/PackageReferenceTest.kt

```diff
@@ -26,14 +26,21 @@ import io.kotlintest.should
 import io.kotlintest.shouldBe
 import io.kotlintest.specs.WordSpec
 
+import java.util.SortedSet
+
 class PackageReferenceTest : WordSpec() {
-    private val node1_1_1 = PackageReference(Identifier.fromString("::node1_1_1"))
-    private val node1_1 = PackageReference(Identifier.fromString("::node1_1"), dependencies = sortedSetOf(node1_1_1))
-    private val node1_2 = PackageReference(Identifier.fromString("::node1_2"))
-    private val node1 = PackageReference(Identifier.fromString("::node1"), dependencies = sortedSetOf(node1_1, node1_2))
-    private val node2 = PackageReference(Identifier.fromString("::node2"))
-    private val node3 = PackageReference(Identifier.fromString("::node3"), dependencies = sortedSetOf(node1_2))
-    private val root = PackageReference(Identifier.fromString("::root"), dependencies = sortedSetOf(node1, node2, node3))
+    companion object {
+        fun pkgRefFromIdStr(id: String, dependencies: SortedSet<PackageReference> = sortedSetOf()) =
```

Comment:
Consider using varargs for the 'dependencies parameter.
Also, maybe the parameter type could become string as well?